tests/e2e/stepDefinitions/presentationViewerContext.js

```diff
@@ -0,0 +1,40 @@
+const { Given, When, Then } = require('@cucumber/cucumber')
+const { expect } = require('@playwright/test')
+const PresentationViewer = require('../pageObjects/PresentationViewerPage')
+const assert = require('node:assert')
+const { getUserCredentials } = require('../utils/userHelper')
+const Ocis = require('../pageObjects/OcisPage')
+const { uploadFile } = require('../utils/fileHelper')
+
+const presentationViewer = new PresentationViewer()
+const ocis = new Ocis()
+
+Given(
+  'user {string} has uploaded the markdown file {string} using API',
+  async function (user, fileName) {
+    const response = await uploadFile(fileName, user)
+    assert.equal(response.status, 201, 'Failed to upload file!')
```

Comment:
assertion is not required here because `makeApiRequest` already handles the request error
```suggestion
    await uploadFile(fileName, user)
```